### ISSUE TITLE
Flaky Oracle test

### ISSUE THREAD

**Issue by kibertoad** [knex/knex#3225]
# Environment

Knex version: 0.17.0
Database + version: Oracle
Node 6

From CI:

```
 1) oracle | oracledb
       knex.migrate
         knex.migrate.up
           should only run the first migration if no migrations have run:
      AssertionError: expected [] to have a length of 1 but got 0
      + expected - actual
      -0
      +1
      
      at /home/travis/build/tgriesser/knex/test/integration/migrate/index.js:534:38
  From previous event:
      at Builder.Target.then (lib/interface.js:38:24)
      at knex.migrate.up.then (test/integration/migrate/index.js:533:16)
  From previous event:
      at Context.<anonymous> (test/integration/migrate/index.js:530:12)
```

**Comment by kibertoad** [comment]
Observed on Node 10:
```
  1) oracle | oracledb
       knex.migrate
         knex.migrate.status
           should return 0 if code matches DB:
      AssertionError: expected -1 to equal 0
      + expected - actual
      --1
      +0
      
      at /home/travis/build/tgriesser/knex/test/integration/migrate/index.js:160:39
  From previous event:
      at Context.<anonymous> (test/integration/migrate/index.js:159:12)
  From previous event:
```

**Comment by elhigu** [comment]
Thanks. I'll fix those. Probably adding forUpdate() to some query when checking value from DB is enough. There are already other disabled tests which will catch this error.

**Comment by kibertoad** [comment]
Released in 0.20.10.
